FunctionalEquivalence/FunctionalEquivalence.wdl

```diff
@@ -0,0 +1,508 @@
+version 1.0
+
+import "../BenchmarkVCFs/BenchmarkVCFs.wdl" as BenchmarkVCFs
+
+workflow FunctionalEquivalence {
+    input{
+        Array[String] sample_id
+        Array[Boolean] include_in_fe_analysis
+        Array[Boolean] create_roc_plot
+        Array[String] dataset
+        Array[String] replicate_no
+
+        Array[String] confidence_intervals
+
+        Array[String] tool1_vcf
+        Array[String] tool1_vcf_index
+        Array[String] tool2_vcf
+        Array[String] tool2_vcf_index
+
+        Array[String] truth_vcf
+        Array[String] truth_vcf_index
+
+        Array[File]? stratIntervals
+        Array[String]? stratLabels
+
+        File reference
+        String referenceVersion = "hg38"
+        File refIndex
+        File refDict
+        File hapMap
+
+        String tool1_label
+        String tool2_label
+        String? additional_label
+
+        Boolean signed_difference = false
+
+        String f1_script_http_url = "https://github.com/broadinstitute/palantir-workflows/raw/main/FunctionalEquivalence/scripts/f1_evaluation.py"
+        String fe_script_http_url = "https://github.com/broadinstitute/palantir-workflows/raw/main/FunctionalEquivalence/scripts/fe_evaluation.py"
+        String roc_script_http_url = "https://github.com/broadinstitute/palantir-workflows/raw/main/FunctionalEquivalence/scripts/plot_rocs.R"
```

Comment:
Not ideal, because this will always point to the main branch, even if executing from a feature branch. It can be changed to point to a different branch, but that requires manual intervention.

What do people suggest, heredoc? That would make a massive WDL file. Alternatively I could refactor the plotting tasks as subworkflows and put the heredocs in there.

Reply:
I moved it to subtasks with heredocs, I think that's most elegant. The issue with including it into the docker is that everyone would have to build and push a new docker image whenever they make any changes to the code, which isn't ideal in my opinion.